### DISCUSSION TITLE
Feeback

### DISCUSSION THREAD

**Discussion by abdulbasitwaqas** [AdnanYousaf813/AutoLogout#1] [General]
Hi dev!
Thank you so much. I was just searching about this and finally I completed my task by following your technique.
Thanks hats up!!!!!!

**Comment by AdnanYousaf813** [comment]
@abdulbasitwaqas  thank you for your feedback.

**Comment by arunkr1321** [comment]
Hi @AdnanYousaf813  - I am unable to run the example App as pods are not getting installed, we get missing PODSPEC Error , please resolve it so that we can run example app and would benefit wider developer community ,

**Reply by AdnanYousaf813** [reply]
Hi @arunkr1321, Thank you for your feedback. I just try again to run example by cloning whole project and its working fine, I think you are just downloading the example folder.
Please try again and do let me know your findings. 
Thanks
